laws/applicative.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const {identity, apply} = require('fantasy-combinators');
+const {ap} = require('../index');
+
+const id = (t) => (eq) => (x) => {
```

Comment:
Yeah, I'm still under sided if they should be required for consistency. I'll remove for now.